kernel/configs/x86_64_kata_kvm_4.19.x

```diff
@@ -2722,6 +2722,7 @@ CONFIG_INTEGRITY=y
 CONFIG_DEFAULT_SECURITY_DAC=y
 CONFIG_DEFAULT_SECURITY=""
 CONFIG_CRYPTO=y
+CONFIG_DRM_FBDEV_LEAK_PHYS_SMEM=n
```

Comment:
@GabyCT Can you add an explanation for why this is required as well. 
For the kernel doc, 
https://github.com/torvalds/linux/blob/master/drivers/gpu/drm/Kconfig#L113
Its recommended to set this as disabled.